Sources/ValidatingProperty.swift

```diff
@@ -254,74 +254,74 @@ public final class ValidatingProperty<Value, ValidationError: Swift.Error>: Muta
 				}
 			}
 	}
-}
 
-/// Represents a decision of a validator of a validating property made on a
-/// proposed value.
-public enum ValidatorOutput<Value, Error: Swift.Error> {
-	/// The proposed value is valid.
-	case valid
+	/// Represents a decision of a validator of a validating property made on a
+	/// proposed value.
+	public enum Decision {
```

Comment:
It is a result of an aggressive action, while status is passive state of a matter.